helm_deploy/hmpps-template-typescript/values.yaml

```diff
@@ -32,10 +32,14 @@ generic-service:
 
   # Environment variables to load into the deployment
   env:
-    NODE_ENV: "production"
-    REDIS_TLS_ENABLED: "true"
-    TOKEN_VERIFICATION_ENABLED: "true"
-    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
+    NODE_ENV: 'production'
+    REDIS_TLS_ENABLED: 'true'
+    TOKEN_VERIFICATION_ENABLED: 'true'
+    APPLICATIONINSIGHTS_CONNECTION_STRING: 'InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/'
+    AUTHENTICATION_WINDOW_TIME_IN_MINUTES: '10'
+    AUTHENTICATION_MAX_REQUESTS: '10'
+    AUTHORISATION_WINDOW_TIME_IN_MINUTES: '1'
+    AUTHORISATION_MAX_REQUESTS: '100'
```

Comment:
maybe a `RATE_LIMITER` prefix would help give context - or maybe you could remove from the helm chart and just let it default in config.ts?